exercises/proverb/src/test/java/ProverbTest.java

```diff
@@ -0,0 +1,84 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class ProverbTest {
```

Comment:
These tests should exactly match the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/proverb/canonical-data.json) which I don't think they do at the moment :)

Reply:
I was taking a look at the canonical data, and I copied the tests I'm using from the [Ruby exercise](https://github.com/exercism/ruby/blob/master/exercises/proverb/proverb_test.rb) which includes the qualifier and diverts a bit from the canonical data. To be fair, it also looks like its the third version of the Proverb exercise for Ruby. Since this is the first version of this exercise for Java, I will only include the canonical tests and leave the qualifier and such for future iterations.